app/src/main/java/ru/vssemikoz/newsfeed/models/NewsItemList.java

```diff
@@ -0,0 +1,15 @@
+package ru.vssemikoz.newsfeed.models;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.List;
+
+public class NewsItemList {
+    @SerializedName("articles")
+    private List<NewsItem> newsItem;
```

Comment:
Why do you called a list by single name? `newsItem` is intended to be a `newsItems` as it's a list of objects.